### PR TITLE
Estimator url prod/dev

### LINKED ISSUE #1121: Use Staging Price API Endpoint for development and PR branches
Link to a different price estimation service based on deployment

* By default: Use the `production` ones
* PR + Develop: Use their staging

For example. It could be implemented in travis by having a logic depending on the branch:
```sh
if (
   branch <> master OR
   is not a tag
) {
   BACEND_URL = 1234
}
```

Not sure what is the cleanest way to do it, but travis allows to set rules by branch. Also that the in the shell script, it exposes information as environemt variable to build this conditional behaviour.

### PR DESCRIPTION
**Pull request by Velenir**
Differentiate between different estimators based on master branch/tag release
Fixes #1121

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr1183--dexreact.review.gnosisdev.com

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,8 @@ before_install:
   # Needed to deploy pull request and releases
   - sudo apt-get -y install python-pip python-dev
   - pip install awscli --upgrade --user
+  # use production url for price estimator on master or tagged
+  - export PRICE_ESTIMATOR_URL=$([[ $TRAVIS_BRANCH = "master" || $TRAVIS_TAG != "" ]] && echo "production" || echo "develop")
```
Comment (anxolin) [inline .travis.yml:15]: Sorry it was not in the description. But can we use production one ALSO in the `release/*` and `hotfix/*` branches?

We want those ones AS CLOSE AS POSSIBLE as production

File: .travis.yml [hunk 2]
```diff
@@ -11,6 +11,13 @@ before_install:
   # Needed to deploy pull request and releases
   - sudo apt-get -y install python-pip python-dev
   - pip install awscli --upgrade --user
+  # use production url for price estimator on master or tagged
+  - export PRICE_ESTIMATOR_URL=$([[
+    $TRAVIS_BRANCH = master || \
+    $TRAVIS_BRANCH = release/* || \
+    $TRAVIS_BRANCH = hotfix/* || \
```
Comment (anxolin) [inline .travis.yml:18]: Great! I guess to make sure it works, we need to merge the thing :) 

We now have an open hotfix, it's a great moment to get this thing merged 👍

File: .travis.yml [hunk 3]
```diff
@@ -11,6 +11,13 @@ before_install:
   # Needed to deploy pull request and releases
   - sudo apt-get -y install python-pip python-dev
   - pip install awscli --upgrade --user
+  # use production url for price estimator on master or tagged
+  - export PRICE_ESTIMATOR_URL=$([[
```
Comment (alfetopito) [inline .travis.yml:15]: That's fancy! I was thinking something way simpler, but hey, glad you are doing this :)

**Pull request merged**